e2e/wdio/core/tests/table.e2e-spec.ts

```diff
@@ -0,0 +1,232 @@
+import { TablePo } from '../pages/table.po';
+import {
+    acceptAlert,
+    checkElementScreenshot,
+    click,
+    getAlertText,
+    getAttributeByName,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    getText,
+    isElementClickable,
+    isElementDisplayed,
+    isEnabled,
+    refreshPage,
+    saveElementScreenshot,
+    scrollIntoView,
+    setValue,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import {
+    alertText,
+    componentExampleArr,
+    dateTestText,
+    paginationTestArr,
+    tableCellArr
+} from '../fixtures/appData/table-content';
+
+describe('Table test suite', function() {
+    const tablePage = new TablePo();
+    const {
+        tableExample, link, tableToolbarExample, button, busyIndicator, dialogContent, inputField, table,
+        tableRow, tableCell, markAllCheckboxes, tableCheckboxesExample, tableSemanticExample, tablePopinExample,
+        clickableTableRow, tableNavigatableRowExample, tablePaginationExample, menuItem, paginationLink, tableResult,
+        linkPrevious, linkNext, tableCustomColumnsExample, inputGroup, dialogValue
+    } = tablePage;
+
+    beforeAll(() => {
+        tablePage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(tableExample);
+    }, 1);
+
+    it('should check clickability links for all examples', () => {
+        for (let i = 0; i < 11; i++) {
+            checkIsLinkClickable(componentExampleArr[i]);
+        }
+    });
+
+    describe('Check Table with Toolbar example', function() {
+
+        it('should check display busy indicator', () => {
+            scrollIntoView(tableToolbarExample);
+            click(tableToolbarExample + button, 1);
+            expect(isElementDisplayed(busyIndicator)).toBe(true, 'busy indicator not displayed');
+        });
+
+        it('should check adding a new item', () => {
+            scrollIntoView(tableToolbarExample);
+            click(tableToolbarExample + button, 2);
+            for (let i = 0; i < 3; i++) {
+                setValue(dialogContent + inputField, 'test', i);
+            }
+            click(dialogContent + button, 1);
+            saveElementScreenshot(tableToolbarExample + table, 'table-toolbar-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder());
+            expect(checkElementScreenshot(tableToolbarExample + table, 'table-toolbar-example-' + getImageTagBrowserPlatform(),
+                tablePage.getScreenshotFolder())).toBeLessThan(5, `element item state mismatch`);
```

Comment:
maybe better to get initial row count, add new item, get new row count and check last row has the provided data?
screenshot is ok, but not many people are able to update them.